packages/serverpod_serialization/lib/src/websocket_messages.dart

```diff
@@ -337,26 +350,30 @@ class MethodStreamMessage extends WebSocketMessage {
   /// If this is null the message is sent to the return stream of the method.
   final String? parameter;
 
-  /// The object to send.
-  final String object;
+  /// The object that was sent.
+  final dynamic object;
 
   /// Creates a new [MethodStreamMessage].
-  MethodStreamMessage(Map data)
+  /// The [object] must be an object processed by the
+  /// [SerializationManager.wrapWithClassName] method.
+  MethodStreamMessage(Map data, SerializationManager serializationManager)
       : endpoint = data['endpoint'],
         method = data['method'],
         connectionId = UuidValueJsonExtension.fromJson(data['connectionId']),
         parameter = data['parameter'],
-        object = data['object'];
+        object = serializationManager.deserializeByClassName(data['object']);
 
   /// Builds a [MethodStreamMessage] message.
+  /// The [object] must be an object processed by the
+  /// [SerializationManager.wrapWithClassName] method.
   static String buildMessage({
     required String endpoint,
     required String method,
     required UuidValue connectionId,
     String? parameter,
-    required String object,
+    required Map<String, dynamic> object,
```

Comment:
Since this is a static method and can be called from anywhere, I suggest passing a `SerializationManager` instance as a parameter. This way, we can use `serializationManager.encodeWithType(...)`, ensuring that the `object` parameter always has the desired structure.

Reply:
Good suggestion! will do that 👍